components/ButtonApi.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react'
+import { Text } from 'react-native'
```

Comment:
Nice that you are not importing TouchableOpacity etc, I also noticed that you do not need to import them when you are declaring them in styled components!